integration_tests/e2e/otp.cy.ts

```diff
@@ -0,0 +1,45 @@
+import IndexPage from '../pages/index'
```

Comment:
Loving these new tests! We may need to update the circleci config to include localstack and gotenberg if we want these tests to run as part of the build.

Reply:
integration tests are running as part of the build already, I had issues with the feature flags logic, so I had to skip the pdf one for now, the containers aren't actually required in the integration test phase, the only requirement is wiremock jar, so I cleaned up now